app/lib/lead_provider_api_specification.rb

```diff
@@ -15,6 +15,10 @@ def self.as_hash(version = CURRENT_VERSION)
   end
 
   def self.spec(version = CURRENT_VERSION)
-    YAML.load_file("swagger/#{version}/api_spec.json")
+    swagger_file = "swagger/#{version}/api_spec.json"
+
+    Preprocessor.new(swagger_file).preprocess!
+
+    YAML.load_file(swagger_file)
```

Comment:
Yeah, bit strange not sure why it was done like that (maybe we used to serve our swagger docs in YAML?), but it works because YAML is a superset of JSON so it will parse it fine